components/grid/basic-item.tsx

```diff
@@ -35,7 +35,7 @@ type ItemLayoutValue = {
   display: string
 }
 const getItemLayout = (val: BreakpointsValue): ItemLayoutValue => {
-  const display = val === 0 ? 'display: none;' : 'display: unset;'
+  const display = val === 0 ? 'display: none;' : 'display: inherit;'
```

Comment:
If set to `unset`, then all flex props will be disabled because `display` is `block` when not set.